tests/testapps/CTCatalog/CTCatalog/Tests/CTCRunTestViewController.mm

```diff
@@ -83,7 +83,7 @@ - (void)drawRect:(CGRect)rect {
 
     // Flips y-axis for our frame
     CGContextSetTextPosition(context, 0.0, 10.0);
-    CTRunDraw(run, context, CFRangeMake(0, 0));
+    CTRunDraw(run, context, CFRangeMake(1, 5));
```

Comment:
>CFRangeMake(1, 5) [](start = 28, length = 17)

probably worth making this and text constants so that they're easier to relate to each other?